backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -128,25 +189,25 @@ Employees
 .........
 
 The result headers are as in the example above - only the contents of
-the "docs" section are different.. A typical "doc" for a person could
+the "document" field are different.. A typical "document" for a person could
 be: ::
 
     {
         "uuid":["cee8800a-983d-41fa-998c-b4557d68ec35"],
         "name":["Anna Bjerre Reidl"],
-        "locations":["PHONE",
-          "21557342",
-          "DAR",
-          "Strandgårdsvej 16, Skallerup Klit, 9800 Hjørring",
-          "EMAIL",
-          "annar@hjorring.dk"],
-        "departments":["IT-Support",
+        "locations":[["PHONE",
```

Comment:
This chunk should be highlighted as JSON.

Reply:
:+1: